src/diffusers/__init__.py

```diff
@@ -42,6 +42,7 @@
     from .schedulers import (
         DDIMScheduler,
         DDPMScheduler,
+        DPMSolverDiscreteScheduler,
```

Comment:
Very cool! we should also add this in the docs in 
`docs/source/api/schedulers.mdx`

Reply:
I've added the auto docs in commit f65012c